### ISSUE TITLE
networkMode=mirrored gives an error

### ISSUE THREAD

**Issue by linux-coding649** [microsoft/WSL#11359]
### Windows Version

Microsoft Windows [Version 10.0.19045.4046]

### WSL Version

2.0.9.0

### Are you using WSL 1 or WSL 2?

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.15.133.1-1

### Distro Version

Kali Linux 2024.1

### Repro Steps

in .wslconfig:
```
[wsl2]
networkMode=mirrored
```
same thing with:
```
[experimental]
networkMode=mirrored
```

### Expected Behavior

I can connect to my HTTP server directly from LAN.
https://learn.microsoft.com/en-us/windows/wsl/networking#mirrored-mode-networking

### Actual Behavior

On startup:
`wsl: Mirrored networking mode is not supported, falling back to NAT networking`

**Comment by zcobol** [comment]
@linux-coding649 mirrored networking is available in Windows 11 only. You're stuck with NAT mode in Windows 10,

**Comment by benhillis** [comment]
Yes this looks by design. The error message could be a lot more clear, I'll look at improving that.

**Comment by chanpreetdhanjal** [comment]
Hi. Can you please collect networking logs by following the instructions below?
https://github.com/microsoft/WSL/blob/master/CONTRIBUTING.md#collect-wsl-logs-for-networking-issues

**Comment by realies** [comment]
try using `networkingMode` instead of `networkMode`
ref: https://learn.microsoft.com/en-us/windows/wsl/networking#mirrored-mode-networking
